experimental/traffic_router_golang/README.md

```diff
@@ -17,4 +17,60 @@
     under the License.
 -->
 
+# Traffic Router 
+
 This is a prototype of Traffic Router's HTTP side in Golang.
+
+# How to build
+
+To get this app running locally:
+
+- Clone this repo
+- Install Golang programming language ([instructions](https://golang.org/doc/install))
+
+
+# Configuration
+
+Sample configuration file(cfg.json) available in traffic_router_golang directory, please add coveragezone files to path specified in cfg.json
+   
+
+# Build
+
+Compile and generate binary:
+
+   - `cd traffic_router_goland`
+   - `go mod download`
```

Comment:
As you pointed out, this should be `go mod vendor`, not `go mod download`.

Reply:
ok, let me modify